integration/data/test_basic_ops.py

```diff
@@ -18,6 +18,8 @@
     VOLUME_NAME, ENGINE_NAME
 )
 
+METRICS_RETRY_COUNTS = 10
+METRICS_RETRY_INTERVAL = 0.1
```

Comment:
Only used in the metric test function, so it's OK to declare there.

Reply:
Removed. Thank you.